artman/cli/main.py

```diff
@@ -525,5 +542,16 @@ def _change_owner(flags, pipeline_name, pipeline_kwargs):
             os.chown(gapic_config_path, user_host_id, group_host_id)
 
 
+def _change_folder_owner(folder, user_host_id, group_host_id):
```

Comment:
Just curious about your choice to use "folder" rather than "directory" in a bunch of places. Is this a convention? Otherwise I would stick to just one term, "directory"

Reply:
Done.